packages/venia-concept/src/components/SearchBar/mapProduct.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Map Magento 2.3.1 schema changes to Venia 2.0.0 proptype shape
+ * to maintain backwards compatibility.
+ */
```

Comment:
Can you add a TODO deprecate and remove here?

Reply:
Comment added. 👍